utils/src/main/java/org/corfudb/utils/lock/LockConfig.java

```diff
@@ -0,0 +1,29 @@
+package org.corfudb.utils.lock;
```

Comment:
in discoveryService the LockConfig is generated using the properties file.  Why do we need both?

Reply:
Property file can be used to supply the needed configuration - for example for testing, or if we want to tune the timeouts. This class has a @Builder, as well as @Default annotation for the fields, in case the properties file is not provided and we don't want to configure anything. Basically we used to have these fields sprinkled around the code as static and now they are available as defaults.